zerver/tests/test_import_export.py

```diff
@@ -1083,9 +1145,61 @@ def test_import_files_from_s3(self) -> None:
+                'INFO:root:Successfully imported <class \'zerver.models.BotStorageData\'> from zerver_botstoragedata.',
+                'INFO:root:Successfully imported <class \'zerver.models.BotConfigData\'> from zerver_botconfigdata.',
+                'INFO:root:Successfully imported <class \'zerver.models.UserPresence\'> from zerver_userpresence.',
+                'INFO:root:Successfully imported <class \'zerver.models.UserActivity\'> from zerver_useractivity.',
+                'INFO:root:Successfully imported <class \'zerver.models.UserActivityInterval\'> from zerver_useractivityinterval.',
+                'INFO:root:Successfully imported <class \'zerver.models.CustomProfileField\'> from zerver_customprofilefield.',
+                'INFO:root:Successfully imported <class \'zerver.models.CustomProfileFieldValue\'> from zerver_customprofilefieldvalue.',
+                'INFO:root:Importing avatars',
+                'INFO:root:Importing uploaded files',
+                'INFO:root:Importing emojis',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Importing realm icons and logos',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Uploaded by ID mapped user: 11!',
+                'INFO:root:Successfully rendered Markdown for message batch',
+                'INFO:root:Successfully imported <class \'zerver.models.Message\'> from zerver_message.',
+                'INFO:root:Successfully imported <class \'zerver.models.UserMessage\'> from zerver_usermessage[1].',
+                'INFO:root:Successfully imported <class \'zerver.models.Reaction\'> from zerver_reaction.',
+                'INFO:root:Successfully imported <class \'zerver.models.Attachment\'> from zerver_attachment.',
+                'INFO:root:Successfully imported M2M table zerver_attachment_messages',
+                'INFO:root:Successfully imported <class \'analytics.models.RealmCount\'> from analytics_realmcount.',
+                'INFO:root:Successfully imported <class \'analytics.models.UserCount\'> from analytics_usercount.',
+                'INFO:root:Successfully imported <class \'analytics.models.StreamCount\'> from analytics_streamcount.'
+            ])
+            self.assertEqual(m.output[39][:33] + m.output[39][-33:], 'INFO:root:Importing message dump /test-export/messages-000001.json')
+            self.assertEqual(m.output[44][:41] + m.output[44][-28:], 'INFO:root:Importing attachment data from /test-export/attachment.json')
+            self.assertEqual(m.output[47][:40] + m.output[47][-27:], 'INFO:root:Importing analytics data from /test-export/analytics.json')
```

Comment:
@chdinesh1089 Logs here are almost same but few of them are different, those in line from 1181-1189 are not present in `check_logs` function logs.

Reply:
Yes, it looks right to me. Instead of sending the test name as a parameter, maybe we could send a parameter like `include_uploaded_by` as the lines from 1181-1189 seems to have that?